### ISSUE TITLE
Exclusion rule not working

### ISSUE THREAD

**Issue by BlazDT** [AdguardTeam/AdguardBrowserExtension#1922]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Issue Details

- **AdGuard version:**
  - AdGuard v4.0.64
- **Browser and version:**
  - Chrome 96
- **Operating system and version:**
  -  Windows 10
- **Filters you use in AdGuard:**
  - Base Filter, German filter

### Expected Behavior

Unblock rule working on `https://www.np-coburg.de/`

### Actual Behavior

URL gets blocked
![image](https://user-images.githubusercontent.com/34266271/145291340-a41e3db7-a2a7-478a-bb65-19fe25ae7c20.png)

### Additional Information

Rule `@@||_.rocks/iqadcontroller.js` is in German filter (and also in user filter), but Base filter still blocks it

**Comment by maximtop** [comment]
We have decided to postpone this issue as domains with underscores in the name are not fully valid and a quite a rare case.
If new issues with domains with underscore appear we return to this matter.
I'm closing this issue, feel free to reopen it.
